integration_tests/pages/page.ts

```diff
@@ -191,7 +191,7 @@ export default abstract class Page {
   signOut = (): PageElement => cy.get('[data-qa=signOut]')
 
   private checkHeading(): void {
-    cy.get('.govuk-heading-l').contains(this.pageHeading)
+    cy.get('h1').first().contains(this.pageHeading)
```

Comment:
I was going along the lines of it technically is allowed but I suppose it's best not to, so happy to remove `.first` as it will prevent any potential cases where multiple are on the same page in the future.